models/organization.go

```diff
@@ -0,0 +1,218 @@
+package models
+
+import (
+	"errors"
+	"time"
+
+	"github.com/jinzhu/gorm"
+	"github.com/lib/pq"
+)
+
+// Organization represents a company using white-labeled Temporal
```

Comment:
Why not just call it white label? Organization have a more generic meaning.

Reply:
Ideally the white-labelling system can be used for any type of billing. In this case white-labelling is being used to implement an organization based billing system.